### PR TITLE
Document BMP388 / BMP390

### PR DESCRIPTION
**Pull request by martgras**
## Description:

Document BMP3xx component

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#2716

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/bmp3xx.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+BMP388 / BMP390 Temperature+Pressure Sensor
+===========================================
+
+.. seo::
+    :description: Instructions for setting up BMP388 or BMP390 temperature and pressure sensors with ESPHome
+    :image: bmp388.jpg
+    :keywords: BMP388 BMP390
+
+The ``bmp3xx`` sensor platform allows you to use your BMP388 or BMP390 
+(`datasheet <https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp390-ds002.pdf>`__, `BMP390 product page <https://www.bosch-sensortec.com/products/environmental-sensors/pressure-sensors/bmp390/>`__) temperature and pressure sensors with ESPHome. The :ref:`I²C <i2c>` is
```
Comment (oxan) [inline components/sensor/bmp3xx.rst]:
```suggestion
(`datasheet <https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp390-ds002.pdf>`__, `BMP390 product page <https://www.bosch-sensortec.com/products/environmental-sensors/pressure-sensors/bmp390/>`__) temperature and pressure sensors with ESPHome. The :ref:`I²C <i2c>` bus is
```
(I would also wrap this after the links).

File: components/sensor/bmp3xx.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        pressure:
+          name: "Outside Pressure"
+        address: 0x77
+        update_interval: 60s
+
+Configuration variables:
+------------------------
+
+- **temperature** (*Optional*): The information for the temperature.
+  sensor
```
Comment (oxan) [inline components/sensor/bmp3xx.rst]:
```suggestion
- **temperature** (*Optional*): The information for the temperature sensor.
```

File: components/sensor/bmp3xx.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- **iir_filter** (*Optional*): Set up an Infinite Impulse Response filter to increase accuracy. One of
+  ``OFF``, ``2x``, ``4x``, ``16x``, ``32``, ``64x``, ``128x``. Defaults to ``OFF``.
+- **update_interval** (*Optional*, :ref:`config-time`): The interval to check the
+  sensor. Defaults to ``60s``.
+
+.. figure:: images/bmp388.jpg
+    :align: center
+    :width: 60.0%
+
+    Pins on the BMP388/BMP390. Only VCC, GND, SCL, and SDA need to be connected.
```
Comment (oxan) [inline components/sensor/bmp3xx.rst]: Repeating this image here seems pointless.

File: components/sensor/bmp3xx.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+(`datasheet <https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp390-ds002.pdf>`__, `BMP390 product page <https://www.bosch-sensortec.com/products/environmental-sensors/pressure-sensors/bmp390/>`__) temperature and pressure sensors with ESPHome. The :ref:`I²C <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/bmp388.jpg
+    :align: center
+    :width: 50.0%
+
+    BMP388/BMP390 Temperature and Pressure Sensor.
+
+.. _Adafruit: https://www.adafruit.com/product/2651
```
Comment (oxan) [inline components/sensor/bmp3xx.rst]: Stray link.

Comment (martgras) [reply components/sensor/bmp3xx.rst]: Thanks for the improvements - implemented

**Review by oxan — APPROVED** [review]

File: components/sensor/bmp3xx.rst [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+-  ``8x``
+-  ``16x`` (default)
+-  ``32x``
+
+See Also
+--------
+
+- :ref:`sensor-filters`
+- :doc:`bme280`
+- :doc:`bmp280`
```
Comment (jesserockz) [inline components/sensor/bmp3xx.rst:80]: Duplicate lines
```suggestion
- :doc:`bme280`
```

Comment (oxan) [reply components/sensor/bmp3xx.rst:80]: They're different, BM**E** vs BM**P** (I almost commented as well :P)

Comment (jesserockz) [reply components/sensor/bmp3xx.rst:80]: :facepalm: I read it multiple times, but only the number...haha

**Pull request merged**